Dockerfile-dev

```diff
@@ -0,0 +1,36 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/ziplinedev .
+#
+# To run the container:
+#
+#    docker run -v /path/to/your/notebooks:/projects -v ~/.zipline:/root/.zipline -p 8888:8888/tcp --name zipline -it quantopian/ziplinedev
+#
+# To access Jupyter when running docker locally (you may need to add NAT rules):
+#
+#    https://127.0.0.1
+#
+# default password is jupyter.  to provide another, see:
+#    http://jupyter-notebook.readthedocs.org/en/latest/public_server.html#preparing-a-hashed-password
+#
+# once generated, you can pass the new value via `docker run --env` the first time
+# you start the container.
+#
+# You can also run an algo using the docker exec command.  For example:
+#
+#    docker exec -it zipline zipline run -f /projects/my_algo.py --start 2015-1-1 --end 2016-1-1 /projects/result.pickle
+#
+FROM quantopian/zipline
+
+ADD . /zipline
+WORKDIR /zipline
+RUN pip install -r etc/requirements_dev.txt
+RUN pip install datashape
```

Comment:
We probably want to do `pip install -r etc/requirements_blaze.txt` so that we get the needed version.